front/lib/models/assistant/conversation.ts

```diff
@@ -371,6 +371,14 @@ AgentMessageFeedback.init(
       type: DataTypes.INTEGER,
       allowNull: false,
     },
+    agentConfigurationId: {
+      type: DataTypes.STRING,
+      allowNull: true,
+    },
+    agentConfigurationVersion: {
+      type: DataTypes.INTEGER,
+      allowNull: true,
+    },
```

Comment:
why do we replace the relation ?

Reply:
Because we'll be keeping feedbacks on non agentConfiguration conversations as well (@dust, @gpt, etc)
This will be for our own data, and the ones linked to assistants will be for the builders